src/wazuh_modules/vulnerability_detector/wm_vuln_detector.c

```diff
@@ -8620,4 +8637,15 @@ int wm_vuldet_get_software(int agent_id, bool not_triaged, cJSON** requested_ite
     return result;
 }
 
+int wm_vuldet_init_queue() {
+    vu_tasks_queue = linked_queue_init(vu_tasks_queue_push_callback, vu_tasks_queue_pop_callback);
+
+    if (!vu_tasks_queue) {
+        mwarn("Unable to create the tasks queue for vulnerability detector.");
```

Comment:
If this queue can't be initialized, the module will not work at all. Not sure whether we should log an error instead. Or, in any case log a warning but return a value that would allow the caller void continue running the main loop of Vulnerability Detector.

Reply:
The method is already returning an error code.